src/why3prop.ml

```diff
@@ -0,0 +1,122 @@
+(** Translation of lambdapi goals to why3 goals in propositional case *)
+
+open Terms
+open Extra
+
+exception NoGoalTranslation
+
+(* a type to present the list of why3 constants and lambdapi terms *)
```

Comment:
Improve comment (map from LP terms to Why3 symbols).

Reply:
Fixed in 31871fd86d5e6863e533d8d4323eaaa186a1d9c1